### DISCUSSION TITLE
Configuration to check if Someine is playing Movies with Jellyfin

### DISCUSSION THREAD

**Discussion by grumat** [languitar/autosuspend#429] [Show and tell]
I was looking for a way to keep my server on while someone is watching movies on Jellyfin service. These are the steps:

First you require an API KEY.  See [these instructions](https://quickbox.io/knowledge-base/v2/applications/autoscan/create-an-api-key-for-autoscan-in-jellyfin/) to create a ApiKey.

Then add to your **autosuspend.conf**:
```
[check.JellyFin]
class = JsonPath
enabled = True
url = http://127.0.0.1:8096/Sessions?ApiKey=<paste-here-your-api-key>
jsonpath = $$.[*].NowPlayingItem.Id
```

Please look at the `url` line. There you should add your API Key. Hope it helps.

**Comment by languitar** [comment]
If you have some time, it would be great to transform this into a pull request for the documentation.

**Comment by malvinas2** [comment]
Not sure, if there is a difference between the syntax used by Emby or its fork Jellyfin, but IMHO it should be: 
```
[check.Emby]
class = JsonPath
enabled = true
url = http://<your_ip>:8096/Sessions?api_key=<your_key>
jsonpath = $.[*].NowPlayingItem.Id
```

**Reply by languitar** [reply]
There shouldn't be a difference as the json path is evaluated in autosuspend and not on the server.

**Reply by malvinas2** [reply]
The two differences are: 
`api_key` instead of `ApiKey`, as the URL with apikey doesn't exist. 
One `$`-sign, instead of two `$$`, at least the online available jsonpath-testers didn't recognize the later expression.
